pkgs/by-name/ve/verovio/package.nix

```
@@ -0,0 +1,67 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  cmake,
+  git,
+  makeWrapper,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "verovio";
+  version = "4.2.0";
+
+  src = fetchFromGitHub {
+    owner = "rism-digital";
+    repo = "verovio";
+    rev = "version-${finalAttrs.version}";
+    hash = "sha256-PjBtnt46pHV3rDlwltVxfUMlgNoZGnI9CfnFtk2/j3s=";
```

```suggestion
    hash = "sha256-x9YcDnYG4S0kwtXwXJo223yEIqSZPoZCbbpsM5Po04M=";
```
If possible, avoid leaveDotGit = true. Perhaps the git use in CMake can be patched out.